.github/ISSUE_TEMPLATE/breaking-change.yaml

```
@@ -0,0 +1,40 @@
+name: "Breaking change report"
+description: "Report a breaking change in Elastic Security"
+title: "[BREAKING CHANGE] "
+labels: "breaking-change"
+body: 
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to report a breaking change in Elastic Security software to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team. We will add it to the release notes for the version that introduces the breaking change.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: What feature will break? Any more details about how it will break? Please include any recommendations for users who were using the feature with the breaking change.
+    validations:
+      required: true
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which deployment types does this breaking change apply to?
```

Optional nit, slightly more concise:
```suggestion
      label: Which deployment types are affected?
```